packages/@expo/cli/package.json

```diff
@@ -110,11 +110,15 @@
     ]
   },
   "peerDependencies": {
+    "@expo/router-core": "*",
```

Comment:
Users won't install this package directly, so this won't be fulfilled. We can only really import it if we re-export with a top-level file or folder from `expo-router`